### DISCUSSION TITLE
After the Google service withdrew this, translation plugins were affected.

### DISCUSSION THREAD

**Discussion by whxuezhe** [Alex313031/Thorium-Win-AVX2#61] [Q&A]
Currently only ImTranslator Part of the translation function can be used (only word-marking translation can be used normally), but the entire webpage cannot be translated; I mean, since the thorium builds Teli and the Chromium builds an ecology, can it develop its own translation service? This is a survey for domestic market research The guest is a very helpful gesture; thank you!

**Comment by gz83** [comment]
> Currently only ImTranslator Part of the translation function can be used (only word-marking translation can be used normally), but the entire webpage cannot be translated; I mean, since the thorium builds Teli and the Chromium builds an ecology, can it develop its own translation service? This is a survey for domestic market research The guest is a very helpful gesture; thank you!

@whxuezhe Google service withdraw and the translation function cannot be used normally? It seems that you are a user from mainland China. As far as I know, using Google's services in mainland China will be restricted, but it is believed that users in mainland China should still be able to solve this problem through various translation websites and translation extensions.

At present, the translation function carried by Thorium browser comes from Google official, and Thorium browser implement related functions  through Google API and key.

The development of the Thorium browser's own translation service you mentioned is very meaningful to me personally, but I cannot decide whether it will be realized in the end.
